meinberlin/apps/projects/signals.py

```diff
@@ -0,0 +1,49 @@
+from django.core.cache import cache
+from django.db.models.signals import post_save
+from django.db.models.signals import pre_delete
+from django.dispatch import receiver
+
+from adhocracy4.dashboard import signals as a4dashboard_signals
+from adhocracy4.projects.models import Project
+from meinberlin.apps.projects.tasks import get_next_project_ends
+from meinberlin.apps.projects.tasks import get_next_projects_start
+
+
+@receiver(a4dashboard_signals.project_created)
+@receiver(a4dashboard_signals.project_published)
+@receiver(a4dashboard_signals.project_unpublished)
+def post_dashboard_signal_delete(sender, project, user, **kwargs):
+    cache.delete_many(
+        [
+            "projects_active*",
+            "projects_future*",
+            "projects_past*",
+            "private_projects",
+            "extprojects",
+        ]
+    )
+
+
+@receiver(post_save, sender=Project)
+@receiver(pre_delete, sender=Project)
+def post_save_delete(sender, instance, update_fields=None, **kwargs):
+    """
+    Delete cache for project list views.
+    Capture any new phases that may got created/updated while saving a project.
+    """
+
+    cache.delete_many(
```

Comment:
below and in the docs it says the keys are:
```
       cache.delete_many(
            [
                "projects_activeParticipation",
                "projects_futureParticipation",
                "private_projects",
                "extprojects",
            ]
        )
```
I assume you forgot to change it here ? :)